docs/concepts/local_queue.md

```diff
@@ -4,14 +4,24 @@ A `LocalQueue` is a namespaced object that groups closely related workloads
 belonging to a single tenant. A `LocalQueue` points to one [`ClusterQueue`](cluster_queue.md)
 from which resources are allocated to run its workloads.
 
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: LocalQueue
+metadata:
+  namespace: team-a 
+  name: team-a-lq
+spec:
+  clusterQueue: cluster-queue 
+```
+
 Users submit jobs to a `LocalQueue`, instead of to a `ClusterQueue` directly.
 Tenants can discover which queues they can submit jobs to by listing the
 local queues in their namespace. The command is similar to the following:
 
 ```sh
-kubectl get -n my-namespace localqueues
-# Alternatively, use the alias `queue` or `queues`
```

Comment:
why not leave the aliases?

Reply:
no, it doesn't refer to this command.

It might refer to mentions of `Queue` (with capital letter) instead of `LocalQueue`, like this one:

https://github.com/kubernetes-sigs/kueue/blob/main/docs/tasks/run_jobs.md#0-identify-the-queues-available-in-your-namespace